src/main/java/org/frankframework/frankdoc/model/FrankAttribute.java

```diff
@@ -91,11 +91,11 @@ void handleDefaultExplicitNull(FrankType parameterType) {
 			return;
 		}
 		if(mandatory) {
-			log.warn("Attribute [{}] is mandatory, but it also has a default value: [{}]", toString(), getDefaultValue());
+			log.error("Attribute [{}] is mandatory, but it also has a default value: [{}]", toString(), getDefaultValue());
```

Comment:
Dit lijkt me geen reden om de build te laten falen

Reply:
Zeker, iets mag best verplicht zijn en een default hebben. Of het handig is is een ander verhaal, maar daar hoeft de build niet op te falen.

Reply:
Gedaan.